File: FbxSharpTests/AnimLayerTest.cs
using System;
using NUnit.Framework;
using FbxSharp;

namespace FbxSharpTests
{
    [TestFixture]
    public class AnimLayerTest : TestBase
    {
        [Test]
        public void FbxAnimLayer_Create_HasNamespacePrefix()
        {
            // given:
            var obj = new FbxAnimLayer("asdf");

            // then:
            Assert.AreEqual("AnimLayer::", obj.GetNameSpacePrefix());
        }
    }
}

File: FbxCppTests/AnimLayerTest.cpp

#include "Tests.h"

using namespace std;

void FbxAnimLayer_Create_HasNamespacePrefix()
{
    // given:
    FbxManager* manager = FbxManager::Create();
    FbxAnimLayer* obj = FbxAnimLayer::Create(manager, "asdf");

    // then:
    AssertEqual("AnimLayer::", obj->GetNameSpacePrefix());
}

void AnimLayerTest::RegisterTestCases()
{
    AddTestCase(FbxAnimLayer_Create_HasNamespacePrefix);
}

